test/TestUINodes/TestUINodes.csproj

```diff
@@ -8,16 +8,12 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>TestUINodes</RootNamespace>
     <AssemblyName>TestUINodes</AssemblyName>
-	  <OutputPath>$(OutputPath)\nodes\</OutputPath>
+    <OutputPath>$(OutputPath)\nodes\</OutputPath>
   </PropertyGroup>
-  <ItemDefinitionGroup>
```

Comment:
why is this no longer needed?

Reply:
No longer needed because all test projects will copy their external assembly references (packageReferences) to the test/test_dependencies folder
The new settings are in the CS_SDK.props file